x-pack/plugins/apm/public/components/alerting/fields.tsx

```diff
@@ -43,7 +32,7 @@ export function ServiceField({
 }) {
   return (
     <PopoverExpression
-      value={currentValue || allOption.value}
+      value={currentValue || allOption.label}
```

Comment:
was this change intentional? Why are we adding the label to a value field?

Reply:
Btw. that `allOption` thing is gonna be the end of me. Source of so much confusion.

Reply:
yes it was intentional cause previously we used `ALL_OPTION_VALUE` as value for the allOption.  I updated to https://github.com/elastic/kibana/pull/122648/commits/fbf529e7c80e86c5abeada1573801a103260c0fe 
Hopefully it reduces the confusion